### ISSUE TITLE
Search button in the topbar menu is not displayed on tablet

### ISSUE THREAD

**Issue by ghost** [AlxMedia/hueman#653]
Reported here :
https://wordpress.org/support/topic/search-button-disappeared-on-tablet/#post-9994378

> Hi
> 
> My search button in the top menu isn’t showing on tablet (both portrait and landscape, and I tried several versions of iOS and Android), but in the theme customization tablet preview everything looks normal.
> 
> What’s going on?
> 
> Screenshot of theme customizer: https://prnt.sc/ih8uaz
> Screenshot of actual site on iPad: https://prnt.sc/ih8ufd

**Comment by eri-trabiccolo** [comment]
So the reason, as we know, is:
https://github.com/presscustomizr/hueman/blob/dev/parts/header-nav-main.php#L9
and
https://github.com/presscustomizr/hueman/blob/dev/parts/header-nav-topbar.php#L16
( wp_is_mobile() )
but the question is why we want that :D

**Comment by ghost** [comment]
I don't know. Let's handle this search icon visibility with CSS instead of PHP

**Comment by eri-trabiccolo** [comment]
So this was already handled in CSS, though there was also a logic related to `wp_is_mobile()` which I have now removed.
I've compared the theme resulting from the PR to the current version and I've tested they behave the same way.

Anyway I've spotted a particular case where the current theme version (the PR **doesn't** change this behavior) behaves not completely correctly, or at least is confusing.
That's the case when:
a) we display both the topbar and the header menu in mobiles with logo centered and the search box is set to be displayed in the (desktop) header menu only => the search box **IS NOT** displayed in the mobile header menu
b) we display both the topbar and the header menu in mobiles with logo centered and the search box is set to be displayed in the (desktop) topbar menu only => the search box **IS** displayed in the mobile header menu

So there's this inconsistency plus the fact that the search box displaying related options aforementioned refer to the Destkop devices...
